test/unit/app.js

```diff
@@ -179,8 +179,9 @@ describe('App', function () {
     it('works when app is configured', function () {
       configHelpers.createConfig.call(this)
       configHelpers.registerClient.call(this)
-      const app = new App(this.basePath)
+      this.config.persist() // the config helper does not persist it
 
+      const app = new App(this.basePath)
```

Comment:
I usually separate the *Given* / *When* / *Then* steps in my tests (except for very short ones), hence the `const app` assignment grouped with `configHelpers` calls since both are part of the context (aka *Given*), while the `const app` assignment is the actual subject (aka *When*).